test/unit/rules/require-input-label-test.js

```diff
@@ -63,6 +63,15 @@ generateRuleTests({
     '<input type="hidden"/>',
     '<Input type="hidden" />',
     '{{input type="hidden"}}',
+
+    {
+      config: { allow: ['CustomLabel'] },
+      template: '<CustomLabel><input /></CustomLabel>',
+    },
+    {
+      config: { allow: [/web-label/] },
+      template: '<web-label><input /></web-label>',
+    },
   ],
 
   bad: [
```

Comment:
Let's add one invalid test case with a config but not the matching label.

Reply:
done